fl4health/clients/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Note: This is commented out until logging issues resolved
+# Import client classes so that they can be accessed directly from fl4health.clients
+# from fl4health.clients.apfl_client import ApflClient
+# from fl4health.clients.basic_client import BasicClient
+# from fl4health.clients.clipping_client import NumpyClippingClient
+# from fl4health.clients.constrained_fenda_client import ConstrainedFendaClient
+# from fl4health.clients.ditto_client import DittoClient
+# from fl4health.clients.ensemble_client import EnsembleClient
+# from fl4health.clients.evaluate_client import EvaluateClient
+# from fl4health.clients.fed_pca_client import FedPCAClient
+# from fl4health.clients.fed_prox_client import FedProxClient
+# from fl4health.clients.fedper_client import FedPerClient
+# from fl4health.clients.fedpm_client import FedPmClient
+# from fl4health.clients.fedrep_client import FedRepClient
+# from fl4health.clients.fenda_client import FendaClient
+# from fl4health.clients.fenda_ditto_client import FendaDittoClient
+# from fl4health.clients.flash_client import FlashClient
+# from fl4health.clients.instance_level_dp_client import InstanceLevelDpClient
+# from fl4health.clients.model_merge_client import ModelMergeClient
+# from fl4health.clients.moon_client import MoonClient
+# from fl4health.clients.mr_mtl_client import MrMtlClient
+# from fl4health.clients.nnunet_client import NnunetClient
+# from fl4health.clients.partial_weight_exchange_client import PartialWeightExchangeClient
+# from fl4health.clients.perfcl_client import PerFclClient
+# from fl4health.clients.scaffold_client import ScaffoldClient, DPScaffoldClient
+# from fl4health.clients.tabular_data_client import TabularDataClient
```

Comment:
Yeah exactly. It's less verbose to do `from fl4health.clients import NnunetClient` than `from fl4health.clients.nnunet_client import NnunetClient`. A couple other pretty popular packages do this to make packages seem like a module whilst still being able to split them into separate files. In this instance I think it makes even more sense since most of the modules under client contain a single client class

Reply:
Ah I see. Yeah I'm good with that once the logging stuff is settled!